### DISCUSSION TITLE
Compatibility

### DISCUSSION THREAD

**Discussion by kamalawala** [sanni/cartreader#479] [General Chat]
I don't know about you guys, but I would love to see this thing become THE go-to cartridge dumper/flasher. [dumping.guide](https://dumping.guide) has charts showing off other devices with better overall compatibility (for GB/C/A games, anyway), though it could be outdated.

For the DS family of consoles and Switch, I'm not quite sure since homebrew exists for those, though compatibility would still be nice in my opinion. CD/DVD formats may be best left to being dumped by a computer with an optical drive, or homebrew, since adding an optical drive to this thing might just be clunky and a little silly. As for oddball mediums like the FDS, I'm not quite sure, honestly. What do you think?

I may be able to help out with some code at some point but I'd like to build the dumper first so I can actually test it haha. Also, I'm not familiar with C++. I'm really looking forward to building it, it'll actually be the first thing I've ever... well, made. I've never dealt with electronics like that before, it's pretty exciting. Thoughts on it as a first soldering project? Or is it too risky? Well anyway, I've ordered just about all the parts and I'm going to be getting a soldering station soon too.

**Comment by Mythris29** [comment]
It was my first soldering project and while it was a huge challenge for me I was able to get through it and made a v5 that is working great now. I would recommend with the cheaper parts on ali express to get a few extras since shipping can cost more than the parts in the first place so getting a few more doesn't cost that much more.

**Comment by kamalawala** [comment]
I've received two packages so far; a baggy of 50 slide switches, and an N64 controller port (not the extension). They've arrived surprisingly fast, and as I mentioned I'm very excited. A bit too late to get extras though, unfortunately. However, some of the orders come with extras anyway since they're sold in bulk like the aforementioned slide switches.
